lib/source-maps.js

```diff
@@ -4,14 +4,20 @@ const libSourceMaps = require('istanbul-lib-source-maps')
 const fs = require('fs')
 const path = require('path')
 
-const sourceMapCache = libSourceMaps.createSourceMapStore()
+let sourceMapCache = libSourceMaps.createSourceMapStore()
```

Comment:
with your refactor to stop making new instances of `Nyc`, perhaps you're right that this doesn't need to be a singleton?